File: 1317. Convert Integer to the Sum of Two No-Zero Integers/solution.cs
public class Solution {
    private bool IsNoZero(int n) => !$"{n}".Contains("0");

    public int[] GetNoZeroIntegers(int n) {
        for (var i = 1; i < n; i++) {
            var diff = n - i;
            if (this.IsNoZero(i) && this.IsNoZero(diff)) {
                return new int[]{i, diff};
            }
        }

        return new int[]{};
    }
}

File: 1317. Convert Integer to the Sum of Two No-Zero Integers/solution.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        def isNoZero(n: int) -> bool:
            return f'{n}'.rfind('0') == -1

        for i in range(1, n):
            diff = n - i
            if isNoZero(i) and isNoZero(diff):
                return [i, diff]

        return []
